src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/BulkCombinationPriceType.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Combination;
+
+use Currency;
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\PositiveOrZero;
+use PrestaShopBundle\Form\Admin\Type\TextWithUnitType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\Type;
+
+class BulkCombinationPriceType extends TranslatorAwareType
+{
+    /**
+     * @var Currency
+     */
+    private $defaultCurrency;
+
+    /**
+     * @var string
+     */
+    private $weightUnit;
+
+    /**
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param Currency $defaultCurrency
+     * @param string $weightUnit
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        Currency $defaultCurrency,
+        string $weightUnit
+    ) {
+        parent::__construct($translator, $locales);
+        $this->defaultCurrency = $defaultCurrency;
+        $this->weightUnit = $weightUnit;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('wholesale_price', MoneyType::class, [
+                'required' => false,
+                'label' => $this->trans('Cost price (tax excl.)', 'Admin.Catalog.Feature'),
+                'attr' => ['data-display-price-precision' => self::PRESTASHOP_DECIMALS],
```

Comment:
maybe putting`data-display-price-precision` in a constant or variable would avoid repeating it several times

Reply:
Hmm, you actually gave me another idea, maybe we could integrate this data attribute by default in the prestashop form theme for all MoneyType

Reply:
indeed !

Reply:
@MeKeyCool oops didn't see your message

the idea was not to repeat the same string several time, to avoid mistakes and having to modify each of them if the string must be changed

Reply:
But it should be done in a dedicated PR then